projects/imagemagick/crop_fuzzer.cc

```diff
@@ -0,0 +1,22 @@
+#include <cstdint>
+
+#include <Magick++/Blob.h>
+#include <Magick++/Image.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  if ((sizeof(uint16_t) * 2) < 4) {
```

Comment:
That looks weird :) I guess you meant `if (Size < 2 * sizeof(uint16_t))`

Reply:
Well. That's moderately embarrassing.